finetune.py

```diff
@@ -32,7 +32,7 @@ def cache_hiddens(model, dataloader):
     return cached_hiddens
 
 
-def kl_div(student_hiddens, teacher_hiddens, temp):
```

Comment:
Request: if temperature is not used, please remove it altogether.

Rationale: this saves memory. For models like gemma, creating another tensor of logprobs just to divide by temperature is extremely wasteful with vocabulary size 256k.

Reply:
Nice catch, for Cohere it is also a big problem.